src/axom/quest/examples/quest_marching_cubes_example.cpp

```diff
@@ -1740,12 +1791,18 @@ int main(int argc, char** argv)
     exit(retval);
   }
 
+  axom::utilities::raii::AnnotationsWrapper annotation_raii_wrapper(
+    params.annotationMode);
+  AXOM_ANNOTATE_SCOPE("quest marching cubes example");
```

Comment:
This affects the whole scope, yet is far from the open-brace.

Reply:
I commented about this elsewhere on the thread, but it's probably worth mentioning a second time -- `AXOM_ANNOTATE_SCOPE` doesn't annotate the portion of the scope before it is called. It adds an RAII variable whose constructor begins an annotation region and whose destructor, which is called at the end of the scope, ends the annotation region.

Reply:
Yes, thanks for your explanation.  I'm fine with leaving AXOM_ANNOTATE_SCOPE where it is.